scripts/gulp/dev-server.js

```diff
@@ -3,19 +3,70 @@
 const fs = require('fs');
 const log = require('fancy-log');
 const urlParser = require('url');
+const Mustache = require('mustache');
 
 const { createServer, useSsl } = require('./create-server');
 
-function readmeText() {
-  return fs.readFileSync('./README.md', 'utf-8');
+const DOCUMENT_PATH = './test-documents/';
+
+/**
+ * Render some content in an HTML document with `<pre>` tags
+ *
+ * @param {string} filename - source to inject between `<pre>` tags
+ * @return {string}
+ */
+function preformattedContent(filename) {
+  const textContent = fs.readFileSync(filename, 'utf-8');
+  const template = fs.readFileSync(`${DOCUMENT_PATH}preformatted.txt`, 'utf-8');
```

Comment:
From a quick look at the metadata for the various syntax highlighting plugins available for Mustache, it seems that `.mustache` is the preferred extension.